modules/light-clients/08-wasm/types/msgs_test.go

```diff
@@ -54,27 +54,3 @@ func TestMsgStoreCode_ValidateBasic(t *testing.T) {
 		}
 	}
 }
-
-func TestMsgStoreCode_GetSigners(t *testing.T) {
```

Comment:
can add a test that checks for the address using `Codec`s `GetMsgV1Signers` if people think it would be beneficial.

Reply:
sure, sounds good!